static/external/manyplot.js

```diff
@@ -223,7 +224,7 @@ function createPlotter( canvas, multiFrame, options ) {
 
     // set data to use for plotting; updates plot bounds from data
     plotter.setData = function( dataPairs ) {
```

Comment:
We also could have sorted `dataPairs` before passing it into `setData`, but I don't really see an advantage/disadvantage (unless there is some obscure logging case where we want to sort as early as possible... not likely though).  Seems fine to me to do it here.